### PR TITLE
CEDS-3157 Add link to confirmation page

### PR DESCRIPTION
**Pull request by MaciejRewera**

### CODE REVIEW

File: test/views/UploadYourFilesConfirmationSpec.scala [hunk 1]
```diff
@@ -74,10 +74,25 @@ class UploadYourFilesConfirmationSpec extends DomAssertions with ViewMatchers wi
...
+        view.getElementsByTag("h3").first() must containMessage(s"$pagePrefix.section1.title")
+      }
+
+      "displays the current user's verified email address" in {
+        view.getElementById("verifiedEmail") must containText(email)
+      }
+
+      "has links to the secure messages filter page" in {
+        assertContainsLink(view, messages(s"$pagePrefix.section1.paragraph1.linkText"), controllers.routes.InboxChoiceController.onPageLoad().url)
+        assertContainsLink(view, messages(s"$pagePrefix.section1.paragraph2.linkText"), controllers.routes.InboxChoiceController.onPageLoad().url)
```
Comment (udinobi) [inline test/views/UploadYourFilesConfirmationSpec.scala:92]: same here

File: test/views/UploadYourFilesConfirmationSpec.scala [hunk 2]
```diff
@@ -74,10 +74,25 @@ class UploadYourFilesConfirmationSpec extends DomAssertions with ViewMatchers wi
...
     }
 
-    "have a links to print the page" in {
+    "have a link to print the page" in {
       assertContainsLink(view, messages(s"$pagePrefix.printPage"), "javascript:if(window.print)window.print()")
     }
 
+    "have a paragraph explaining what happens when a query is raised" which {
+      "has a heading" in {
+        view.getElementsByTag("h3").first() must containMessage(s"$pagePrefix.section1.title")
```
Comment (udinobi) [inline test/views/UploadYourFilesConfirmationSpec.scala:83]: Even if it's working as it is now (is it?), I would change it to `${pagePrefix.section1.title}` ?

File: test/views/UploadYourFilesConfirmationSpec.scala [hunk 3]
```diff
@@ -74,10 +74,25 @@ class UploadYourFilesConfirmationSpec extends DomAssertions with ViewMatchers wi
...
+
+      "has links to the secure messages filter page" in {
+        assertContainsLink(view, messages(s"$pagePrefix.section1.paragraph1.linkText"), controllers.routes.InboxChoiceController.onPageLoad().url)
+        assertContainsLink(view, messages(s"$pagePrefix.section1.paragraph2.linkText"), controllers.routes.InboxChoiceController.onPageLoad().url)
+      }
+    }
+
     "have a paragraph explaining next steps" which {
       "has a heading" in {
         view.getElementsByTag("h3").get(1) must containMessage(s"$pagePrefix.section2.title")
```
Comment (udinobi) [inline test/views/UploadYourFilesConfirmationSpec.scala:98]: same here

File: test/views/UploadYourFilesConfirmationSpec.scala (hunk 2 above)
Comment (MaciejRewera) [reply test/views/UploadYourFilesConfirmationSpec.scala:83]: I think you are missing the contet here. `pagePrefix` is a field defined at the top of this file.
But it confused me as well, so I'll change it so there is no misunderstanding anymore.

**Review by udinobi — APPROVED** [review]

**Pull request merged**